assets/src/components/editing/toolbar/buttons/CommandButton.tsx

```diff
@@ -16,11 +17,16 @@ export const CommandButton = (props: Props) => {
 
   return (
     <button
-      className={classNames(styles.toolbarButton, active?.(editor) && styles.active)}
+      className={classNames(
+        'disabled:opacity-50',
+        styles.toolbarButton,
+        active?.(editor) && styles.active,
```

Comment:
There appear to still be a hover style applied even though the  button is disabled, I think if possible we should remove the hover background change as well when disabled is true.

I think to accomplish this you need to add a CSS rule in `assets/src/components/editing/toolbar/Toolbar.modules.scss` line 89, something like

```
  &[disabled]:hover {
    background-color: transparent;
  }

```